test/e2e/operator/deployment_api.go

```diff
@@ -744,6 +769,84 @@ var _ = deploy.DescribeForSome("API", func(d *deploy.Deployment) bool {
 			}
 		})
 	})
+
+	Context("conversion", func() {
+		getAlphaDeployment := func(name string) alphaapi.Deployment {
+			return alphaapi.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: name,
+				},
+				Spec: alphaapi.DeploymentSpec{
+					Image: dummyImage,
+				},
+			}
+		}
+		It("with default values", func() {
+			alphaDep := getAlphaDeployment("alpha-default-values")
+			deploy.CreateAlphaDeploymentCR(f, alphaDep)
+
+			deployment := deploy.GetDeploymentCR(f, alphaDep.Name)
+
+			defer deploy.DeleteDeploymentCR(f, deployment.Name)
+
+			validateDriver(deployment, true)
+
+			// Expect to ruturn alpha object converting from stored beta CR
```

Comment:
I missed this when we merged the PR, but now I am looking at a test failure in this test after fixing the default log level and have some more questions.

Why does it say here "from stored beta CR"? The stored CR is in the v1alpha1 format, isn't it?

Reply:
Stored version is always the latest(`spec.storage: true` in the CRD) version. And when the conversion happens to client requested version from that.